### PR TITLE
Changing templates to target .NET 4.6

### PR DESCRIPTION
**Pull request by jwallra**

### CODE REVIEW

File: templates/ProjectTemplate/VisualBasic/1033/MobileAppsApplication.vbhtml.tt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<#@ template hostspecific="true" language="C#" #>
+﻿net46<#@ template hostspecific="true" language="C#" #>
```
Comment (pvbchak) [inline templates/ProjectTemplate/VisualBasic/1033/MobileAppsApplication.vbhtml.tt]: copy-paste ?

**Comment by pvbchak** [comment]
Change v$targetframeworkversion$ in MobileAppsApplication.tt#22 to "v4.6"

File: templates/ProjectTemplate/VisualBasic/1033/MobileAppsApplication.vbhtml.tt (hunk 1 above)
Comment (jwallra) [inline templates/ProjectTemplate/VisualBasic/1033/MobileAppsApplication.vbhtml.tt]: whoops, will fix

**Comment by pvbchak** [comment]
LGTM

**Pull request merged**